gluten-celeborn/pom.xml

```diff
@@ -26,6 +26,16 @@
       <artifactId>celeborn-client-spark-${spark.major.version}-shaded_${scala.binary.version}</artifactId>
       <version>${celeborn.version}</version>
       <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.celeborn</groupId>
+          <artifactId>celeborn-client-spark-${spark.major.version}_${scala.binary.version}</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.celeborn</groupId>
+          <artifactId>celeborn-spark-${spark.major.version}-columnar-shuffle_${scala.binary.version}</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
the shaded artifacts should not expose the shipped transitive deps, seems there are issues in 0.4.0-incubating cc @cfmcgrady

Reply:
@pan3793  I am handling this issue.